lib/datadog/appsec/context.rb

```diff
@@ -55,6 +55,22 @@ def run_rasp(type, persistent_data, ephemeral_data, timeout = WAF::LibDDWAF::DDW
         result
       end
 
+      def mark_as_interrupted!
+        @interrupted = true
+      end
+
+      def interrupted?
+        @interrupted == true
```

Comment:
I think it's better to have default value as `false` in the initializer instead. Undefined instance variables make it harder to follow the code logic.

Reply:
fixed in [f4de7ed](https://github.com/DataDog/dd-trace-rb/pull/4839/commits/f4de7edb068f62f5413a2678b9197d6063015c30)

Reply:
```suggestion
        @interrupted
```